apps-rendering/src/adSlot.tsx

```diff
@@ -117,11 +117,15 @@ const AdSlot: FC<Props> = ({ className, paragraph, format }): ReactElement => (
 		key={`ad-after-${paragraph}-para`}
 	>
 		<div css={adLabelsStyles(format)} className="ad-labels">
-			<h1>Advertisement</h1>
+			<p>
+				<strong>Advertisement</strong>
+			</p>
 		</div>
 		<div css={adSlotStyles} className="ad-slot"></div>
 		<div css={supportBannerStyles(format)} className="support-banner">
-			<h1>Support the Guardian and enjoy the app ad-free.</h1>
+			<p>
+				<strong>Support the Guardian and enjoy the app ad-free.</strong>
```

Comment:
Does this text need to be a `strong`? Could this just be considered the body copy for the `aside` section, and therefore ok in a `p` tag?

Reply:
I don't see why not. I remembered you mentioning a `strong` tag for this but perhaps I misunderstood